mu4/extensions/qml/MuseScore/Extensions/ExtensionItem.qml

```diff
@@ -0,0 +1,137 @@
+import QtQuick 2.7
+import QtQuick.Layouts 1.15
+
+import MuseScore.UiComponents 1.0
+import MuseScore.Extensions 1.0
+
+Rectangle {
+    id: root
+
+    color: ui.theme.popupBackgroundColor
+    radius: 12
+
+    property string code: ""
+    property alias name: nameLabel.text
+    property string description: ""
+    property int status: 0
+    property bool selected: false
+
+    signal clicked(string code)
+
+    RowLayout {
+        anchors.fill: parent
+
+        Item {
+            width: parent.width / 2.1
+            height: parent.height
+            clip: true
+
+            Rectangle {
+                anchors.fill: parent
+                anchors.rightMargin: -12 // NOTE: for a rounded corner on the left side only
+                radius: root.radius
+                border.width: root.border.width
+                border.color: root.border.color
+
+                color: "#595959" // TODO
+
+                StyledTextLabel {
+                    anchors.fill: parent
+
+                    text: qsTrc("extensions", "Placeholder")
+                    font.pixelSize: 30
+                }
+            }
+        }
+
+        Item {
+            Layout.fillHeight: true
+            Layout.fillWidth: true
```

Comment:
readonly property int margin: 58

and use that in nameLabel and descriptionLabel